project/Build.scala

```diff
@@ -173,6 +173,10 @@ object MacroGLBuild extends MechaRepoBuild {
       "-optimise",
       "-feature"
     ),
+    libraryDependencies ++= Seq(
```

Comment:
This should not be needed, normally the LWJGL plugin resolves those dependencies.